### PR TITLE
Fix OCP popup close triggers

### LINKED ISSUE #1458: OCP: Menu vanishes when trying to select destination blog
It makes it impossible (or annoying) to reblog to any blog except for the default one. Seems to only happen on Firefox.

### PR DESCRIPTION
**Pull request by puxlit**
This fixes #1458. It also permits dismissal of the popup after interacting with its inputs by clicking outside the popup (to defocus the input). It won't dismiss the popup if we tab between inputs and our mouse happens to be outside the popup.

### CODE REVIEW

File: Extensions/one_click_postage.js [hunk 1]
```diff
@@ -696,18 +696,17 @@ XKit.extensions.one_click_postage = new Object({
 		};
 
 		var menu_close = function() {
-			// Only close the menu if it doesn't have keyboard or mouse focus
-			if ($("#x1cpostage_box").find('input:focus, textarea:focus').length === 0 &&
-				$('#x1cpostage_box:hover').length === 0) {
```
Comment (nightpool) [inline Extensions/one_click_postage.js]: why remove the hover selector? This seems to say that as soon as you deselect one text area, the menu will close.

Comment (puxlit) [reply Extensions/one_click_postage.js]: In practice, at least with Firefox, `#x1cpostage_box:hover` will match the popup whilst we're handling `mouseleave`. Your options for defocussing any of the inputs are: (i) focus on another input; (ii) click on the fake radio control; or (iii) click on one of the trigger buttons. Handlers exist to cancel the timeout for the first two cases, and the third case is a non-issue.

Comment (nightpool) [reply Extensions/one_click_postage.js]: debounce?

Comment (puxlit) [reply Extensions/one_click_postage.js]: That's what the original timer mechanism is for.

**Comment by puxlit** [comment]
@nightpool bump

**Comment by puxlit** [comment]
@nightpool bump

**Comment by puxlit** [comment]
Any updates, @nightpool? It's been two months, during which at least half a dozen users have taken the time to bring up this issue in the support channel.

CC: @hobinjk

**Pull request merged**